src/components/Modals/LoginModal/LoginModal.js

```diff
@@ -33,27 +41,34 @@ function LoginModal(props) {
       openAnotherModal={props.openAnotherModal}
       formRef={formRef}
       isButtonActive={isButtonActive}
+      onSubmit={submit}
     >
       <label className="modal__label"><p className="modal__label-text">Email</p>
         <input className="modal__input"
           type="text"
-          id="email"
+          id="login-email"
           placeholder="Enter email"
-          onChange={toggleButtonState}
+          onChange={(e) => {
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.

This is how React controls inputs.To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components